### PR TITLE
Fix 'uninitialized constant Net::LDAP::PDU::LdapPduError' error

### PR DESCRIPTION
**Pull request by unagi**
This PR **fixes 'uninitialized constant Net::LDAP::PDU::LdapPduError** when unknown pdu-type occured.

### CODE REVIEW

File: test/test_ldap_connection.rb [hunk 1]
```diff
@@ -340,6 +340,19 @@ def test_queued_read_bind_sasl
     assert result.success?
     assert_equal 2, result.message_id
   end
+
+  def test_invalid_pdu_type
+    options = {
+      app_tag: Net::LDAP::PDU::SearchResult,
```
Comment (satoryu) [inline test/test_ldap_connection.rb]: Is this required? not used in this test case.

File: lib/net/ldap/pdu.rb [hunk 2]
```diff
@@ -123,7 +123,7 @@ def initialize(ber_object)
     when ExtendedResponse
       parse_extended_response(ber_object[1])
     else
-      raise LdapPduError.new("unknown pdu-type: #{@app_tag}")
+      raise Net::LDAP::PDU::Error, "unknown pdu-type: #{@app_tag}"
```
Comment (satoryu) [inline lib/net/ldap/pdu.rb]: Thanks!

File: test/test_ldap_connection.rb (hunk 1 above)
Comment (unagi) [reply test/test_ldap_connection.rb]: That's right!
I remove this parameter.

**Comment by unagi** [comment]
@mtodd @satoryu  ping

**Pull request merged**